client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -31,17 +31,22 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 
 		return (
 			<div className="steps__main">
-				<DocumentHead title={ translate( 'Summary ‹ Jetpack Onboarding' ) } />
+				<DocumentHead title={ translate( 'Summary ‹ Jetpack Start' ) } />
```

Comment:
:disappointed: 17 existing translations will be lost with this change.

Reply:
Well, sucks to be you, @a8ci18n bot :trollface: